ingest/ingest_pipeline.py

```diff
@@ -202,7 +190,12 @@ def close_matrix(self):
         """Closes connection to file"""
         self.matrix.close()
 
+    # @profile
+    def insert_many(self, documents):
```

Comment:
I think that we should probably also include the collection name for this, rather than hard-coding it to `data_arrays`.  For now, that is the only place we call `insert_many()`, but I have a feeling that it will change.

Reply:
Done in 5f53c3e.